float64 local_x
float64 local_y
uint8 height
uint8 az_angle
uint8[] channels
